app/src/main/java/com/android/solvit/shared/ui/chat/ChatScreen.kt

```diff
@@ -101,6 +134,139 @@ fun ChatScreen(
       }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AiSolverWelcomeScreen(navigationActions: NavigationActions) {
+  Scaffold(
+      topBar = {
+        TopAppBar(
+            title = {},
+            navigationIcon = {
+              IconButton(onClick = { navigationActions.goBack() }) {
+                Icon(Icons.Default.ArrowBack, contentDescription = "Back")
+              }
+            },
+        )
+      }) { innerPadding ->
+        BoxWithConstraints(
+            modifier =
+                Modifier.fillMaxSize()
+                    .background(color = Color(0xFFF5F5F5))
+                    .padding(innerPadding)
+                    .testTag("AiGetStartedScreen"),
+            contentAlignment = Alignment.Center) {
+              val screenHeight = maxHeight
+
+              Column(
+                  horizontalAlignment = Alignment.CenterHorizontally,
+                  verticalArrangement = Arrangement.SpaceEvenly,
+                  modifier = Modifier.fillMaxHeight()) {
+                    Text(
+                        text =
+                            buildAnnotatedString {
+                              withStyle(
+                                  style =
+                                      SpanStyle(
+                                          color = Color.Black,
+                                          fontSize = screenHeight.times(0.03f).value.sp)) {
+                                    append("Meet Your Personal ")
+                                  }
+                              withStyle(
+                                  style =
+                                      SpanStyle(
+                                          color = Color(0xFF6AA84F),
+                                          fontSize = screenHeight.times(0.03f).value.sp)) {
+                                    append("AI\n")
+                                  }
+                              withStyle(
+                                  style =
+                                      SpanStyle(
+                                          color = Color(0xFF6AA84F),
+                                          fontSize = screenHeight.times(0.03f).value.sp)) {
+                                    append("Problem Solver")
+                                  }
```

Comment:
The code section above seems to be a bit repetitive, is there a way to make a reusable function which will take a string as input?

Reply:
Actually , since it's a build annotated string, it works like this on Kotlin. I agree that it seems to be repeated but it's not too much disturbing I think !